README.md

```diff
@@ -1,2 +1,19 @@
 # outdated
-Application to show when to update packages to the next lts version.
+
+This Application is meant to make it easier to know what Project has outdated Packages, Outdated Packages are Packages that have gone out of Long Term Support (short lts). There is a colour-code to what state a Project is in:
+
+Green: Project is up to date
+
+Yellow: One or more Packages released a new lts version, you "should" migrate to it now since the other version is still in support
+
+Red: One or more Packages are outdated
+
+Black: All Packages are outdated, this is not good
+
+---
+
+## How to start the Application?
+
+First install Docker-Compose and Poetry, after that download or clone this project, then rename .env.dist file and change the values of the variables to your liking. Then run ```docker-compose up -d``` this will start the database.
```

Comment:
Please use the newer syntax of docker compose: `docker compose up -d` without the dash